examples/generated/test/react-dom/interactivity/AccessibilityTest.js

```diff
@@ -4,10 +4,72 @@ import styled from "styled-components"
 import colors from "../colors"
 import shadows from "../shadows"
 import textStyles from "../textStyles"
+import createActivatableComponent from "../utils/createActivatableComponent"
 
 export default class AccessibilityTest extends React.Component {
+  state = { focusRing: false }
+
+  setFocusRing = (focusRing) => { this.setState({ focusRing }) }
+
+  focus = ({ focusRing = true } = { focusRing: true }) => {
+    this.setFocusRing(focusRing)
+
+    let focusElements = this._getFocusElements()
+    if (focusElements[0] && focusElements[0].focus) {
+      focusElements[0].focus()
+    }
+  }
+
+  focusNext = ({ focusRing = true } = { focusRing: true }) => {
+    this.setFocusRing(focusRing)
+
+    let focusElements = this._getFocusElements()
+    let nextIndex = focusElements.indexOf(document.activeElement) + 1
+
+    if (nextIndex >= focusElements.length) {
+      this.props.onFocusNext && this.props.onFocusNext()
+      return ;
+    }
+
+    focusElements[nextIndex].focus && focusElements[nextIndex].focus()
+  }
+
+  focusPrevious = ({ focusRing = true } = { focusRing: true }) => {
```

Comment:
I think we should probably declare now that people will need to have a way to transform class properties in their consuming app.

Reply:
Good thinking. I think it's OK since anybody using JSX likely should be set up for this, but we can also go with a constructor at some point if needed